source/getting-started/securing.markdown

```diff
@@ -13,9 +13,9 @@ One of the reasons to use Home Assistant is that it's not depending on cloud ser
 
 ### {% linkable_title Checklist %}
 
-- [Protect you web interface with a password](https://home-assistant.io/getting-started/basic/#password-protecting-the-web-interface)
+- [Protect your web interface with a password](https://home-assistant.io/getting-started/basic/#password-protecting-the-web-interface)
 - Secure your host. Sources could be [Red Hat Enterprise Linux 7 Security Guide](https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/pdf/Security_Guide/Red_Hat_Enterprise_Linux-7-Security_Guide-en-US.pdf), [CIS Red Hat Enterprise Linux 7 Benchmark](https://benchmarks.cisecurity.org/tools2/linux/CIS_Red_Hat_Enterprise_Linux_7_Benchmark_v1.0.0.pdf), or the [Securing Debian Manual](https://www.debian.org/doc/manuals/securing-debian-howto/index.en.html).
-- Restrict network access to your device. Set `PermitRootLogin no` and to use keys for authentication instead of passwords.
+- Restrict network access to your device. Set `PermitRootLogin no` in your sshd config (usually `/etc/ssh/sshd_config`) and to use keys for authentication instead of passwords.
```

Comment:
Yes, this would be most likely already be covered by the first point but better mention this twice as the hardening guides are pretty extensive.